components/financial_assistance/app/views/financial_assistance/applications/application_checklist.html.erb

```diff
@@ -7,7 +7,7 @@
       <div class="col-lg-10 col-md-10 col-sm-10 col-xs-12 row">
 
         <input type='hidden' name='exit_after_method' value='false' id='exit_after_method'>
-        <div class="col-lg-9 col-md-9 col-sm-9 col-xs-9 bottom-pd">
+        <div class="col-lg-9 col-md-9 col-sm-9 col-xs-9 bottom-pd mb-20">
```

Comment:
This additional class should be removed as we no longer are displaying on this page so this is unnecessary